smart-contracts/migrations/3_create_proxy.js

```diff
@@ -1,23 +1,46 @@
-const shell = require('shelljs')
+// Load zos scripts and truffle wrapper function
+const { scripts, ConfigVariablesInitializer } = require('zos')
+const { create } = scripts
 
-module.exports = function deployProxy (deployer, network, accounts) {
+async function deploy (options, accounts) {
   // default account used by ganache
   const unlockOwner = accounts[0]
+
+  // Create an instance of MyContract, setting initial value to 42
```

Comment:
Please delete or update the comments here

Reply:
Yup.